config/config.go

```diff
@@ -153,13 +153,38 @@ func (cfg *Config) validateServicePipelines() error {
 
 // Service defines the configurable components of the service.
 type Service struct {
+	Telemetry ServiceTelemetry
+
 	// Extensions are the ordered list of extensions configured for the service.
 	Extensions []ComponentID
 
 	// Pipelines are the set of data pipelines configured for the service.
 	Pipelines Pipelines
 }
 
+// ServiceTelemetry defines the configurable settings for service telemetry.
+type ServiceTelemetry struct {
+	Logs ServiceTelemetryLogs `mapstructure:"logs"`
+}
+
+// ServiceTelemetryLogs defines the configurable settings for service telemetry logs.
+// This MUST be compatible with zap.Config. Cannot use directly zap.Config because
+// the collector uses mapstructure and not yaml tags.
+type ServiceTelemetryLogs struct {
```

Comment:
Maybe check in `Validate` that the `Level` and `Encoding` fields have a valid value?

Reply:
Nice suggestion, done.